index.html

```diff
@@ -1,30 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>Save The Bay</title>
-
-    <!-- Latest compiled and minified CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
-
-    <link rel="stylesheet" href="styles/main.css" />
-  </head>
-  <body>
-    <div class="container">
-      <div class="jumbotron">
-        <h1>Save the Bay</h1>
-      </div>
+<head>
+  <title>Save the Bay</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
```

Comment:
I like to have lines no longer than 80 characters to make sure they don't scroll off the screen on smaller screens.  Can we commit to using that practice in our files?  If so, let's update this.